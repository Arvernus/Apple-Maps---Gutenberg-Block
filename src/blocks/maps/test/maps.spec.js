import { createNewPost, insertBlock, getEditedPostContent } from '@wordpress/e2e-test-utils';

describe( 'maps', () => {
	beforeEach( async () => {
		await createNewPost();
	} );

	test( 'block post content stays the same', async () => {
		await insertBlock( 'Mapkit' );
		expect( await getEditedPostContent() ).toMatchSnapshot();
	} );
} );
