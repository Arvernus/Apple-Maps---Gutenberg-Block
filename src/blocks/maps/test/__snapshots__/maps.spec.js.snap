// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`maps block post content stays the same 1`] = `"<!-- wp:mapkitjs/map /-->"`;

exports[`maps works 1`] = `"<!-- wp:mapkitjs/map /-->"`;
